doc/classes/Node.xml

```diff
@@ -364,6 +364,12 @@
 				If [param include_internal] is [code]false[/code], returns the index ignoring internal children. The first, non-internal child will have an index of [code]0[/code] (see [method add_child]'s [code]internal[/code] parameter).
 			</description>
 		</method>
+		<method name="get_internal_mode" qualifiers="const">
+			<return type="int" enum="Node.InternalMode" />
+			<description>
+				Returns the node's internal mode. See [enum InternalMode].
```

Comment:
Worth mentioning when this value is written to. You know as well as me it's in `add_child()`/`add_sibling()`.